packages/allure-hermione/src/index.ts

```diff
@@ -0,0 +1,318 @@
+    });
+  };
+
+  hermione.on(hermione.events.NEW_BROWSER, (browser) => {
+    browser.addCommand("label", async (testId: string, name: string, value: string) => {
+      await addLabel(testId, name, value);
+    });
+    browser.addCommand(
+      "link",
+      async (testId: string, url: string, name?: string, type?: string) => {
+        await addLink(testId, url, name, type);
+      },
+    );
+    browser.addCommand(
+      "parameter",
+      async (testId: string, name: string, value: string, options?: ParameterOptions) => {
+        await addParameter(testId, name, value, options);
+      },
+    );
+    browser.addCommand("epic", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.EPIC, value);
+    });
+    browser.addCommand("feature", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.FEATURE, value);
+    });
+    browser.addCommand("story", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.STORY, value);
+    });
+    browser.addCommand("suite", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.SUITE, value);
+    });
+    browser.addCommand("parentSuite", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.PARENT_SUITE, value);
+    });
+    browser.addCommand("subSuite", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.SUB_SUITE, value);
+    });
+    browser.addCommand("owner", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.OWNER, value);
+    });
+    browser.addCommand("severity", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.SEVERITY, value);
+    });
+    browser.addCommand("tag", async (testId: string, value: string) => {
+      await addLabel(testId, LabelName.TAG, value);
+    });
+    browser.addCommand("issue", async (testId: string, name: string, url: string) => {
+      await addLink(testId, url, name, LinkType.ISSUE);
+    });
+    browser.addCommand("tms", async (testId: string, name: string, url: string) => {
+      await addLink(testId, url, name, LinkType.TMS);
+    });
+    browser.addCommand("attach", async (testId: string, source: string, mimetype: string) => {
+      await addAttachment(testId, source, mimetype);
+    });
+  });
+  hermione.on(hermione.events.NEW_WORKER_PROCESS, (worker) => {
+    // eslint-disable-next-line
+    // @ts-ignore
+    worker._process.on("message", (message) => {
+      if (message?.type === ALLURE_METADATA_CONTENT_TYPE) {
+        const { testId, metadata } = message as HermioneAttachmentMessage;
+
+        handleAllureAttachment(testId, metadata);
+      }
+    });
+  });
+  hermione.on(hermione.events.TEST_BEGIN, (test) => {
+    const { ALLURE_HOST_NAME, ALLURE_THREAD_NAME } = process.env;
+    const thread = ALLURE_THREAD_NAME || test.sessionId;
+    const hostname = ALLURE_HOST_NAME || os.hostname();
```

Comment:
`os.hostname()` cache this value in some variable as we do in the other frameworks?